Pipfile

```
@@ -19,7 +19,7 @@ pysha3="*"
 requests = "*"
 sqlalchemy = "*"
 apistar = "==0.5.42"
-tzlocal = "==2.0.0b1"
+tzlocal = "==1.5.1"
```

```suggestion
tzlocal = "<2.0.0"
```

I mean... I'll make it.  :-)

---

Also in `setup.py`. And, if it doesn't break stuff (@KPrasch was saying that it does)

---

Leaving this until later... worried about keeping a good set a dependencies pre-cedar